SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -119,5 +116,20 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	/** The deployment to connect to when using the Development Authentication Flow. If left empty, it uses the first available one (order not guaranteed when there are multiple items). The deployment needs to be tagged with 'dev_login'. */
 	UPROPERTY(EditAnywhere, config, Category = "Cloud Connection", meta = (ConfigRestartRequired = false))
 	FString DevelopmentDeploymentToConnect;
-};
 
+	/** Single server worker type to launch when offloading is disabled, fallback server worker type when offloading is enabled (owns all actor classes by default). */
+	UPROPERTY(EditAnywhere, Config, Category = "Offloading")
+	FWorkerType DefaultWorkerType;
+
+	/** Enable running different server worker types to split the simulation by Actor Groups. */
+	UPROPERTY(EditAnywhere, Config, Category = "Offloading")
+	bool bEnableOffloading;
+
+	/** Actor Group configuration. */
+	UPROPERTY(EditAnywhere, Config, Category = "Offloading", meta = (EditCondition = "bEnableOffloading"))
+	TMap<FName, FActorGroupInfo> ActorGroups;
+
+	/** Available server worker types. */
+	UPROPERTY(Config)
+	TSet<FName> WorkerTypes;
```

Comment:
I think we should rename this `ServerWorkerTypes` as this it what it is. @mattyoung-improbable @Vatyx @alastairdglennie @tenevdev  What's your thoughts?

Reply:
I agree it's a better name

Reply:
I agree. I'll go ahead and change the WorkerTypes in launch config to be ServerWorkerTypes too.